sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1079,6 +1088,15 @@ public virtual AsyncPageable<T> QueryAsync<T>(
         /// <param name="filter">
         /// Returns only entities that satisfy the specified filter expression.
         /// For example, the following expression would filter entities with a PartitionKey of 'foo': <c>e => e.PartitionKey == "foo"</c>.
+        /// <para>
+        /// The following string comparison methods are supported as part of a filter expression:
+        /// <list type="bullet">
+        /// <item><see cref="string.Equals(string)"/></item>
+        /// <item><see cref="string.Equals(string, string)"/></item>
+        /// <item><see cref="string.CompareTo(string)"/></item>
+        /// <item><see cref="string.Compare(string, string)"/></item>
```

Comment:
```suggestion
        /// <item><description><see cref="string.Equals(string)"/></description></item>
        /// <item><description><see cref="string.Equals(string, string)"/></description></item>
        /// <item><description><see cref="string.CompareTo(string)"/></description></item>
        /// <item><description><see cref="string.Compare(string, string)"/></description></item>
```

_([docs](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags#list))_

We'll need the same for the others. lest MS Learn view them as malformed and refuse to render them.